diff -Nur -x '*.orig' -x '*.rej' e-smith-proftpd-1.12.0/root/var/service/proftpd/log/run mezzanine_patched_e-smith-proftpd-1.12.0/root/var/service/proftpd/log/run
--- e-smith-proftpd-1.12.0/root/var/service/proftpd/log/run	2007-04-06 12:02:45.000000000 -0600
+++ mezzanine_patched_e-smith-proftpd-1.12.0/root/var/service/proftpd/log/run	2007-04-06 12:00:15.000000000 -0600
@@ -17,9 +17,7 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307  USA
 #----------------------------------------------------------------------
-chown smelog:smelog /var/log/proftpd 	# [SME: 1267]
-
 exec					\
     /usr/local/bin/setuidgid smelog	\
     /usr/local/bin/multilog t s5000000	\
-    /var/log/proftpd
+    /var/log/ftp
