diff -Nur -x '*.orig' -x '*.rej' e-smith-proftpd-1.12.0/root/var/service/proftpd/log/run mezzanine_patched_e-smith-proftpd-1.12.0/root/var/service/proftpd/log/run
--- e-smith-proftpd-1.12.0/root/var/service/proftpd/log/run	2006-03-17 11:09:50.000000000 +1100
+++ mezzanine_patched_e-smith-proftpd-1.12.0/root/var/service/proftpd/log/run	2006-04-18 13:39:34.000000000 +1000
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 #----------------------------------------------------------------------
-# copyright (C) 2003 Mitel Networks Corporation
+# copyright (C) 2003-2006 Mitel Networks Corporation
 # 
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
@@ -16,10 +16,9 @@
 # You should have received a copy of the GNU General Public License
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307  USA
-# 
-# Technical support for this program is available from Mitel Networks 
-# Please visit our web site www.mitel.com/sme/ for details.
 #----------------------------------------------------------------------
+chown smelog:smelog /var/log/proftpd 	# [SME: 1267]
+
 exec					\
     /usr/local/bin/setuidgid smelog	\
     /usr/local/bin/multilog t s5000000	\
